BUILD.md

```diff
@@ -38,18 +37,6 @@ notes below.
     - [Record Data for AutoFDO Builds](#record-data-for-autofdo-builds)
     - [Build Using AutoFDO Data](#build-using-autofdo-data)
 
-## Requirements
-
-- **Windows newer than XP**
-  - **NTFS-based C:**, because msys2 doesn't work on FAT filesystems
-- **MacOS** 10.x
-- **Haiku** up-to-date
-- **Ubuntu** 16.04 or newer
-- **Fedora** up-to-date
-- **RedHat or CentOS** 7 or newer
-- **Arch-based distribution** up-to-date
-- **OpenSUSE Leap** 15 or newer
-
```

Comment:
Makes sense.

Regarding PRs for old and less popular systems, we could give them some heads with a statement like:

_"Rare and/or end-of-life operating system are not tested. Your mileage may vary, but fee free to try and report back to us."_

I was tempted to say "Not support" and other languages - but we don't want to scare people away from even trying.  If they report back, we might see a tiny change that will "flip the switch" for them; or we might see it plagued with issues in which case the answer will be definitive "sorry, we're can't help".

Reply:
I replaced this with:
```
## Minimum build requirements

- C/C++ compiler with support for C++14
- SDL >= 2.0.2
- Meson >= 0.49.0 or Visual Studio Community Edition
- OS that is mostly POSIX-compliant or up-to-date Windows system

All other dependencies are optional and can be disabled while configuring the
build (in `meson setup` step).
```

Because I don't want to maintain a list of specific OSes that we support or not. DOSBox Staging can be built on every OS that's either POSIX or Windows and has the necessary modern tools available.